cirq-core/cirq/ops/classically_controlled_operation_test.py

```diff
@@ -473,7 +473,8 @@ def test_decompose():
     op = cirq.H(q0).with_classical_controls('a')
     assert cirq.decompose(op) == [
         (cirq.Y(q0) ** 0.5).with_classical_controls('a'),
-        cirq.XPowGate(exponent=1.0, global_shift=-0.25).on(q0).with_classical_controls('a'),
+        cirq.X(q0).with_classical_controls('a'),
```

Comment:
Yeah, so here's where we probably need to have an opt in flag on the decompose context object. Having the basic gates suddenly decompose to multiple gates by default probably would be too likely to break something. While generally it seems that changing details around complex decompositions have been approved, this change seems a little too fundamental.